packages/docs/docs/video.md

```diff
@@ -11,7 +11,7 @@ Videos with audio are not supported - yet! Audio will be muted in the final rend
 
 ## API / Example
 
-Use an import or require to load an audio file and pass it as the `src` prop. All the props that the native `<video>` element accepts (except `autoplay` and `controls`) will be forwarded (but of course not all are useful for Remotion). This means you can use all CSS to style the video.
+Use an import or require to load an video file and pass it as the `src` prop. All the props that the native `<video>` element accepts, `startAt` (optional) and `endAt` (optional) props (except `autoplay` and `controls`) will be forwarded (but of course not all are useful for Remotion). This means you can use all CSS to style the video.
```

Comment:
Nice! Makes it sound like `startAt` prop will be forwarded to the video element, which shouldn't happen. Can we leave this sentence and then mention `startAt` in a new paragraph? It will probably also be easier to read and digest for the reader.

Reply:
done